resources/views/v2/components/FormSearchField/FormSearchField.css

```diff
@@ -0,0 +1,102 @@
+@import "variables";
+
+.FormSearchField {
+    display: flex;
+    align-items: center;
+    position: relative;
+    width: 100%;
+}
+
+    .FormSearchField .FormSearchField__input {
+        -webkit-transition: all 0.15s linear;
+        -moz-transition: all 0.15s linear;
+        -o-transition: all 0.15s linear;
+        transition: all 0.15s linear;
+    }
+
+
+    .FormSearchField__input {
+        background: rgba(255, 255, 255, .2);
+        box-shadow: 0 18px 48px rgba(14, 18, 23, .2);
+        outline: 0;
+        -webkit-appearance: none;
+        font: $font-heading-md;
+        padding: 5px 40px $padding-xxs 50px;
+        color: $white;
+        border: 0;
+        width: 100%;
+        height: 50px;
+    }
+
+    .FormSearchField__input:hover,
+    .FormSearchField__input:focus {
+        color: $black;
+    }
+
+        .FormSearchField:hover .FormSearchField__input,
+        .FormSearchField__input:focus {
+            background: rgb(255, 255, 255);
+        }
+
+        .FormSearchField ::-webkit-input-placeholder { /* WebKit, Blink, Edge */
```

Comment:
Can we collect these somehow or get rid of vendor prefixes totally? Verify the compiled css is autoprefix up the job